BedrockServer.cpp

```diff
@@ -395,7 +395,8 @@ void BedrockServer::sync()
                 continue;
             }
 
-            if (command->shouldPostProcess()) {
+            if (command->shouldPostProcess() && command->response.methodLine == "200 OK") {
```

Comment:
I don't know if this is the best way to decide that this command needs to be post-processed.